src/components/Dashboard/TodoList.vue

```diff
@@ -877,6 +923,12 @@ export default {
       })
     },
 
+    async getBcolMessage (errCode: string): Promise<any> {
+      const fetchUrl = sessionStorage.getItem('PAY_API_URL') + 'codes/errors/' + errCode
+      const errObj = await axios.get(fetchUrl)
+      return errObj.data
```

Comment:
Might consider putting this in a mixin so we can re-use the mixin class between apps easily. Also probably should add some error handling and maybe present some kind of generic message if fetching the error message fails.

Might need to ask Scott what he wants ( 'Details currently unavailable.'?).

Reply:
done

Reply:
Since Ody is away this week, I confirm that this code was satisfactorily moved to a mixin.

However, it looks like there is no fallback if the fetch returns no errorObj. Is this a conscious decision? Or does Scott suggest something here?